sats-dna/src/main/kotlin/com/sats/dna/components/proteinbar/SatsProteinBar.kt

```diff
@@ -112,6 +116,11 @@ fun SatsProteinBar(
     }
 }
 
+typealias ProteinBarDuration = SnackbarDuration
+typealias ProteinBarHostState = SnackbarHostState
+typealias ProteinBarResult = SnackbarResult
```

Comment:
Why not just rename Snackbar* to ProteinBar*?

Reply:
Cause we don't own those. They're part of the Material APIs.

Reply:
As per https://github.com/sats-group/sats-dna-android/blob/main/sats-dna/src/main/kotlin/com/sats/dna/components/proteinbar/SatsProteinBar.kt#L11-L14.